x/withdraw/types/interfaces.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+
+	claimstypes "github.com/tharsis/evmos/v2/x/claims/types"
+)
+
+// BankKeeper defines the banking keeper that must be fulfilled when
+// creating a x/withdraw keeper.
+type BankKeeper interface {
+	IterateAccountBalances(ctx sdk.Context, addr sdk.AccAddress, cb func(coin sdk.Coin) (stop bool))
+	BlockedAddr(addr sdk.AccAddress) bool
+}
+
+// AccountKeeper defines the expected account keeper
+type AccountKeeper interface {
+	GetAccount(sdk.Context, sdk.AccAddress) authtypes.AccountI
+}
+
+// TransferKeeper defines the expected IBC transfer keeper.
+type TransferKeeper interface {
+	GetDenomTrace(ctx sdk.Context, denomTraceHash tmbytes.HexBytes) (transfertypes.DenomTrace, bool)
+	SendTransfer(
```

Comment:
On `interfaces.go` I think methods can go all in one line, regardless of their length, cause we're mostly just checking of the method name is included.